src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -49,6 +49,17 @@
  * @author lkuehne
  */
 public class PackageObjectFactory implements ModuleFactory {
+
+    /**
+     * Enum class to define loading options.
+     */
+    public enum ModuleLoadOption {
+        /** Searching from registred checkstyle modules. **/
+        SEARCH_REGISTERED_PACKAGES,
+        /** As SEARCH_REGISTERED_PACKAGES and also searching for class in checkstyle packages. **/
```

Comment:
> checkstyle packages

This makes it sound like we are only searching checkstyle packages. We are actually searching packages defined in `packages` field supplied through constructor, which is none by default.
Please update.